docs/Project.toml

```diff
@@ -0,0 +1,6 @@
+[deps]
+CCDReduction = "b790e538-3052-4cb9-9f1f-e05859a455f5"
```

Comment:
Remove the package itself from the dependencies, it will be check out from the local directory
```suggestion
```